applications/ChimeraApplication/custom_python/chimera_python_application.cpp

```diff
@@ -0,0 +1,62 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+// ==============================================================================
+//  ChimeraApplication
+//
+//  License:         BSD License
+//                   license: ChimeraApplication/license.txt
+//
+//  Authors:        Aditya Ghantasala, https://github.com/adityaghantasala
+// 					Navaneeth K Narayanan
+//					Rishith Ellath Meethal
+// ==============================================================================
+//
+// System includes
+
+#if defined(KRATOS_PYTHON)
+// External includes
+
+// Project includes
+#include "includes/define_python.h"
+#include "chimera_application.h"
+#include "chimera_application_variables.h"
+#include "custom_python/add_custom_processes_to_python.h"
+#include "custom_python/add_custom_utilities_to_python.h"
+#include "custom_python/add_custom_strategies_to_python.h"
+
+
+namespace Kratos
+{
+
+namespace Python
+{
+
+  using namespace pybind11;
```

Comment:
this gives some problems with namespace clashes, please take a look at the rest of the code to see how it is done no => `using namespace py = pybind11`